### DISCUSSION TITLE
Custom CSS overwritten

### DISCUSSION THREAD

**Discussion by eLGorditoMIA** [darkreader/darkreader#7947] [Q&A] [answered]
Hi All,

First time user and so far loving the tool, my eyes are very thankful :). I am trying to add some custom CSS when using servicenow. Currently when you have a string inside the IDE it puts it as blue, I am tryin to get it to yellow so I added this piece to the dev tools trying to make it show yellow vs blue as I see yellow a bit better. I checked the element and looks like it is running but then it is overwritten by style class "darkreader darkreader--sync"  and that is where I am stuck. 

Anyone know how to force it to use my custom css vs the style class? 

================================

my.service-now.com

CSS
.cm-s-snc span.cm-string {
    color: rgb(255,255,0);
}
.span.cm-string {
    color: rgb(255,255,0);
}
.cm-string {
    color: rgb(255,255,0);
}

================================

**Comment by Gusted** [comment, marked as answer]
>  it is overwritten by style class "darkreader darkreader--sync" and that is where I am stuck.

That's not possible, you might want to look if the fix is properly saved in the devtools(when the formatting is incorrect, it's discarded).

**Reply by eLGorditoMIA** [reply]
Yep you were right, I took a look and as soon as I added !important it worked. thank you for the quick reply.

================================

my.service-now.com

CSS
.cm-s-snc span.cm-string {
color: rgb(255,255,0) !important;
}

================================
